### PR TITLE
si: update translations and minor improvements

### PR DESCRIPTION
**Pull request by hanariyada**
- Fixed: untranslated strings and errors

### CODE REVIEW

**Comment by sahanyada** [comment]
@alexanderby you can use [**zoho forever free plan**](https://www.zoho.com/mail/zohomail-pricing.html) for darkreader.org and other domains.
1 domain/Master account, Up to five users, 30 alias/User, 5GB/User, 25MB attachment limit, No ads.
Free web access, [desktop](https://www.zoho.com/mail/desktop) and [mobile](https://www.zoho.com/mail/mobile) apps
*[alias](https://help.zoho.com/portal/en/kb/mail/adminconsole/articles/how-do-i-add-an-email-alias) - contact@, bugs@, hi@

I noticed that you've configured MX records for darkreader.org, shutau.com but you're still using darkreaderapp@gmail.com :/

**Review by Gusted — APPROVED** [review]
Thanks!

File: src/_locales/si.config [hunk 1]
```diff
@@ -43,73 +43,73 @@ Extension එක සකසන්න
...
-Grayscale
+අළු පරිමාණය
 
 @sepia
-Sepia
+සෙපියා
 
 @only_for
-සඳහා පමණි
+හුදෙක්
```
Comment (Gusted) [inline src/_locales/si.config:91]:
```suggestion
හුදෙක්
```

**Comment by Gusted** [comment]
CC @bershanskiy [flaky test](https://github.com/darkreader/darkreader/actions/runs/3238539906/jobs/5310001368) for MV3?

**Comment by hanariyada** [comment]
Ping @Gusted @bershanskiy

**Comment by wnilanka** [comment]
Can you please check and merge this PR?

**Comment by Myshor** [comment]
Compared to other translations... Dark Reader as a name of extension should not be translated but leaved as it is.

So change back all "ඩාර්ක් රීඩර්" to Dark Reader... or just tell me I can change it back - then I can merge. 😉

**Comment by hanariyada** [comment]
> Compared to other translations... Dark Reader as a name of extension should not be translated but leaved as it is.
> So change back all "ඩාර්ක් රීඩර්" to Dark Reader... or just tell me I can change it back - then I can merge. 😉

Done 😊

**Pull request merged**